django/contrib/postgres/signals.py

```diff
@@ -18,3 +19,20 @@ def register_hstore_handler(connection, **kwargs):
         # This is also needed in order to create the connection in order to
         # install the hstore extension.
         pass
+
+    try:
+        with connection.cursor() as cursor:
+            cursor.execute("SELECT typarray FROM pg_type WHERE typname = 'citext'")
```

Comment:
Is it worth a comment explaining what this is doing? I'm not sure if it's obvious.